frontend/src/plant/usePlants.ts

```diff
@@ -20,5 +20,19 @@ export default function usePlants() {
         () => getAllPlants(), []
     )
 
-    return {plants}
-}
\ No newline at end of file
+    const notify = () => toast("cannot access database");
```

Comment:
This error message is not always true - maybe more generic like "Plant could not be added". You could also show the error message within the toast